### PR TITLE
4.55.0

### PR DESCRIPTION
**Pull request by bwp91**
pre-emptively requesting approval

**to be merged on 2024-01-06** (subject to no sudden bugs arising)

semver minor change --> 4.55.0

---

### UI Changes

- Add plugin log download button
- Improve plugin card hyperlinks (#1918) (@jsiegenthaler)
- Download logs option for when a plugin install/update/uninstall fails
- revise system information icons (os/node)
- accessory title fix (lights + windows) (#1933) (@mkz212)
- improve spacing in plugin card
- more items in homebridge settings
- confirm restart after updating Homebridge instead of automatically restarting
- improve child bridge sing/plu in dropdown
- move backup/restore + users to hb settings
- move hb/ui settings to main menu
- create standard settings page + link at top
- small changes to plugin card (based on feedback)
- fix overflow in markdown changelog
- plugin card updates
- added credits link in status footer and modal for credits
- added more `aria-label` attributes to buttons to improve accessibility
- add link to homebridge wiki in logs on plugin action error
- allow fan/fanv2 rotation speed to be a different unit than percentage
- improve plugin sorting, with actionable plugins at the top
- fix footer paddings for mobile and desktop (#1956) (@mkz212)
- no need to restart if updating disabled plugin
- standardise update modal header

### Translation Changes

- updates to pl.json language file (#1929) (@mkz212)
- updates to de.json language file (@Staubgeborener)
- updates to fr.json language file (@SonnyT72)
- updates to de.json language file (#1953) (@Staubgeborener)
- updates to pl.json language file (#1954) (@mkz212)

### Other Changes

- add `passport` as dependency to BE
- revert ngx-monaco-editor dependency update
- ui: update to angular v15
- improve github actions
- Fix for backup service including npm's cache (#1949) (@sapireli)
- updated dependencies
- ignore `.venv` in hb backup files
- updated Discord Webhooks so notifications are seperated for `release` and `pre-release`
- update maximum file upload size to 25mb

### CODE REVIEW

**Review by donavanbecker — APPROVED** [review]

**Review by donavanbecker — DISMISSED** [review]

File: .vscode/settings.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "editor.insertSpaces": true,
   "files.eol": "\n",
   "editor.codeActionsOnSave": {
-    "source.fixAll.eslint": true
+    "source.fixAll.eslint": "explicit"
```
Comment (NorthernMan54) [inline .vscode/settings.json:6]: Didn't this trigger the await issue earlier this year ?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -1,26 +1,31 @@
...
 
 on:
-  # Removed publish on push, and forced manual release from github
-  #  push:
-  #    branches: [latest]
-  #    tags:
-  #      - '*.*.*'
+  push:
+    branches: 
+      - latest
```
Comment (NorthernMan54) [inline .github/workflows/release.yml]: Will this trigger a release with every push to latest ?

Comment (donavanbecker) [reply .github/workflows/release.yml]: This change will resolve this: https://github.com/homebridge/homebridge-config-ui-x/commit/85773600473fb013058e7004c84b3a0fdf0f215b

File: .vscode/settings.json (hunk 1 above)
Comment (donavanbecker) [reply .vscode/settings.json:6]: I believe @bwp91 resolved that.

**Pull request closed without merging**